prio/server.c

```diff
@@ -83,6 +83,19 @@ PrioServer_aggregate(PrioServer s, PrioVerifier v)
   return MPArray_addmod(s->data_shares, arr, &s->cfg->modulus);
 }
 
+SECStatus
+PrioServer_merge(PrioServer s, const_PrioServer s_i)
+{
+  if (s->cfg->num_data_fields != s_i->cfg->num_data_fields ||
```

Comment:
For extra safety, it might be a good idea to check that `server_a_pub` and `server_b_pub` are the same in both configs.

Reply:
Also, this code should check that `s->idx == s_i->idx`.